test/BaseTokenWrapper.t.sol

```diff
@@ -92,10 +93,17 @@ abstract contract BaseTokenWrapperTest is Test {
       1,
       'Unexpected ending aToken balance'
     );
+
+    assertEq(
+      tokenOut.allowance(address(tokenWrapper), pool),
+      0,
+      'Unexpected TOKEN_OUT allowance'
+    );
```

Comment:
Might be overkill, but we could also check that the tokenWrapper has 0 allowance to the `TokenOut` contract as well, since it's now reset in the intermediary call to `_wrapTokenIn()`

Reply:
That part belongs to each implementation, so I d prefer to do not add it here but in each token wrapper test if necessary